client/src/controllers/CountController.ts

```diff
@@ -0,0 +1,64 @@
+import { Controller } from '@hotwired/stimulus';
+import { ngettext } from '../utils/gettext';
+
+export class CountController extends Controller<HTMLFormElement> {
+  static targets = ['total', 'label'];
+  static values = {
+    containerBody: { type: String, default: 'body' },
+    min: { type: Number, default: 0 },
+    target: {
+      type: String,
+      default:
+        '.error-message,.help-critical,[aria-invalid="true"], .error, .w-field--error',
+    },
+    total: Number,
+  };
+
+  declare minValue: number;
+  declare totalValue: number;
+  declare containerValue: string;
+  declare targetValue: string;
+
+  declare readonly hasTotalTarget: string;
+  declare readonly totalTarget: HTMLInputElement;
+  declare readonly totalTargets: HTMLInputElement[];
+
+  declare readonly hasLabelTarget: string;
+  declare readonly labelTarget: HTMLInputElement;
+  declare readonly labelTargets: HTMLInputElement[];
+
+  declare readonly hasActiveClass: string;
+  declare readonly activeClass: string;
+  declare readonly activeClasses: string[];
+
+  connect() {
+    this.count();
+  }
+
+  count() {
+    const containerElement = document.getElementsByTagName(
+      this.containerValue,
+    )[0];
+    const errorElements = containerElement.getElementsByClassName(
+      this.targetValue,
+    );
+    this.totalValue = errorElements.length;
+    return this.totalValue;
+  }
+
+  totalValueChanged() {
+    this.element.classList.add(this.activeClass);
```

Comment:
```suggestion
  totalValueChanged(total) {
    if (this.hasActiveClass) {
        this.element.classList.toggle(this.activeClass, total > 0);
    }
```

A few notes here
* Probably good to be safe when accessing the activeClass - only try to toggle it if set on the element.
* Use toggle to conditionally add or remove the class  https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/toggle
* Read the `total` from the callback params and avoid re-reading it from `this.totalValue` below.